aspnet/tutorials/nano-server.rst

```diff
@@ -97,39 +117,47 @@ In the ``system.webServer`` section group, update the handlers section to allow
   :dedent: 8
   :emphasize-lines: 2
 
-Add ``httpPlatformHandler`` to the ``globalModules`` section
+Add ``AspNetCoreModule`` to the ``globalModules`` section
 
 .. literalinclude:: nano-server/applicationHost.config
   :language: xml
   :lines: 210,224
   :dedent: 8
   :emphasize-lines: 2
 
-Additionally, add ``httpPlatformHandler`` to the ``modules`` section
+Additionally, add ``AspNetCoreModule`` to the ``modules`` section
 
 .. literalinclude:: nano-server/applicationHost.config
   :language: xml
   :lines: 275,286
   :dedent: 8
   :emphasize-lines: 2
 
+Installing .NET Core Framework
+------------------------------
+
+If you published a portable app, .NET Core must be installed on the target machine. Execute the following Powershell script in a remote Powershell session to install the .NET Framework on your Nano Server.
+
+.. literalinclude:: nano-server/Download-Dotnet.ps1
+  :language: powershell
+
 Publishing the application
------------------------------------
+--------------------------
 
-Copy over the published output of your existing application to the Nano server.
+Copy over the published output of your existing application to the Nano server. You may need to make changes to your *web.config* to point to where you extracted ``dotnet.exe``. Alternatively, you can add ``dotnet.exe`` to your path.
```

Comment:
> Copy over the published output of your existing application to the Nano server

It could be helpful to give more info on this as I found it difficult initially when playing with Nano server
For example i do the following:
```
netsh advfirewall firewall set rule group="File and Printer Sharing" new enable=yes
net share musicstore=c:\deployed\musicstore /GRANT:EVERYONE`,FULL
```